### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by xlNightmare**
Adding another reviewer for spanish since he helps me a lot when none of the other two are online and he also has translated some files with me.

### CODE REVIEW

**Comment by Aby3kr** [comment]
Hi, i'm the contributor ^^

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Contributors
...
 - [Korean](https://github.com/mty22/krunker.io-translations/blob/main/kr.js) (한국어) - Translated by [GoonMandu](https://krunker.io/social.html?p=GoonMandu) (apinklover#0933) and [dojin](https://krunker.io/social.html?p=profile&q=dojin.) (dojin#0001)
-- [Spanish](https://github.com/mty22/krunker.io-translations/blob/main/es.js) (Español) - Translated by [nightz](https://krunker.io/social.html?p=profile&q=xlNightmare) (nightz#6678), reviewed by [MGVoltJesk](https://krunker.io/social.html?p=profile&q=MGVoltJesk) (MGVoltJesk#8000) and [cesar](https://krunker.io/social.html?p=profile&q=AMOLAPIJA) (cesar#9999)
+- [Spanish](https://github.com/mty22/krunker.io-translations/blob/main/es.js) (Español) - Translated by [nightz](https://krunker.io/social.html?p=profile&q=xlNightmare) (nightz#6678), reviewed by [MGVoltJesk](https://krunker.io/social.html?p=profile&q=MGVoltJesk) (MGVoltJesk#8000) , [cesar](https://krunker.io/social.html?p=profile&q=AMOLAPIJA) (cesar#9999) and [Aby3](https://krunker.io/social.html?p=profile&q=Aby3)
```
Comment (mty22) [inline README.md:62]: `(MGVoltJesk#8000) ,` <-- can you please remove the space between `)` and `,`?

**Review by mty22 — CHANGES_REQUESTED** [review]

**Comment by mty22** [comment]
@Aby3kr also ping me in KB for the role~

**Review by mty22 — APPROVED** [review]

**Comment by Vanity8685** [comment]
is their any more grammar issues?

**Pull request merged**